File: output/ifc4x3_dev_73740fe4/entity/IfcObjectPlacement.java
package com.topoplanet.ifc4x3_dev_73740fe4.entity;
import com.topoplanet.test.type.*;


import java.util.Set;

public abstract class IfcObjectPlacement {

    public IfcObjectPlacement PlacementRelTo; // OPTIONAL
    // INVERSE attributes:
    // PlacesObject : SET [0:?] OF IfcProduct FOR ObjectPlacement;
    // ReferencedByPlacements : SET [0:?] OF IfcObjectPlacement FOR PlacementRelTo;

    // SUPERTYPE OF:
    // - IfcGridPlacement
    // - IfcLinearPlacement
    // - IfcLocalPlacement
}

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcObjectPlacement/IfcObjectPlacement.cs
public class IfcObjectPlacement
{
    public IfcObjectPlacement PlacementRelTo { get; set; }

    // === EXTENDED BY ===
    // IfcGridPlacement
    // IfcLinearPlacement
    // IfcLocalPlacement

    // === INVERSE CLAUSES ===
    // PlacesObject : SET [0:?] OF IfcProduct FOR ObjectPlacement
    // ReferencedByPlacements : SET [0:?] OF IfcObjectPlacement FOR PlacementRelTo
}
